application/classes/Ushahidi/Console/ContactImporter.php

```diff
@@ -0,0 +1,140 @@
+<?php defined('SYSPATH') or die('No direct script access');
+
+/**
+ * Ushahidi Webhook Console Command
+ *
+ * @author     Ushahidi Team <team@ushahidi.com>
+ * @package    Ushahidi\Console
+ * @copyright  2014 Ushahidi
+ * @license    https://www.gnu.org/licenses/agpl-3.0.html GNU Affero General Public License Version 3 (AGPL3)
+ */
+
+use Ushahidi\Console\Command;
+use Ushahidi\Core\Entity\UserRepository;
+use Ushahidi\Core\Entity\ContactRepository;
+use Ushahidi\Factory\DataFactory;
+use Ushahidi\Core\Traits\UserContext;
+use Ushahidi\Core\Tool\FormatterTrait;
+
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+use League\Csv\Reader;
+
+
+class Ushahidi_Console_ContactImporter extends Command
+{
+
+	use UserContext;
+	use FormatterTrait;
+
+	private $userRepository;
+	private $contactRepository;
+	private $existing_contacts;
+	private $existing_contacts_lookup = array();
+
+	public function setDataFactory(DataFactory $data)
+	{
+		$this->data = $data;
+	}
+
+	public function setUserRepo(UserRepository $repo)
+	{
+		$this->userRepository = $repo;
+	}
+
+	public function setContactRepo(ContactRepository $repo)
+	{
+		$this->contactRepository = $repo;
+	}
+
+	protected function configure()
+	{
+		$this
+			->setName('contact')
+			->setDescription('Contact functions')
+            ->addArgument('action', InputArgument::OPTIONAL, 'list, import', 'list')
```

Comment:
Could you drop the `action` argument and follow a pattern more like config:set config:set ? ie just implement execute and have the command to a single thing. Makes it easier to migrate to artisan later